docs/PISPLinkingAPISeq.md

```diff
@@ -8,3 +8,12 @@ Example PISP Linking
 ## Happy Path - OTP
 
 ![](../out/docs/sequence/PISPLinkingApi/PISP Linking OTP.svg)
+
+## Error Scenarios
+
+![](../out/docs/sequence/PISPLinkingApi/error_scenarios/PISP Linking OTP Error Scenarios Request Consent Phase.svg)
+![](../out/docs/sequence/PISPLinkingApi/error_scenarios/PISP Linking Web Error Scenarios Request Consent Phase.svg)
```

Comment:
These 2 look the same to me... Am I missing anything?

Reply:
One is for web and one is for OTP.

Reply:
They look the same though - so we might as well amalgamate them if they are identical

Reply:
They are different in the request consent phase part. Instead of having the sequence diagram cover multiple branching flows I've seperated them into their own files due to comments where I would cover different scenarios in the same seq. diagram in the past and people commented that it would be better to have them in their own file.